server/src/main/java/io/unitycatalog/server/service/Scim2UserService.java

```diff
@@ -230,9 +230,9 @@ private HttpResponse handleUserUpdate(String id, PatchOperation operation) {
   private HttpResponse handleExceptionDuringPatch(Exception ex) {
     if (ex instanceof ScimException) {
       throw new Scim2RuntimeException((ScimException) ex);
-    } else   {
+    } else {
       throw new Scim2RuntimeException(
-          new ServerErrorException("Problem with patch operation",ex.getMessage(), ex));
+          new ServerErrorException("Problem with patch operation", ex.getMessage(), ex));
```

Comment:
Undo?

Reply:
This is done automatically by the formatter